build/SandBoxAndPackage.proj

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Layout" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$(RepoRoot)/Directory.Build.props" />
+
+  <PropertyGroup>
+    <SandBoxFolderName>bin/WorkingCopy</SandBoxFolderName>
+    <RelativeSandBoxPackageOutputFolder>$(SandBoxFolderName)/RelativeSandBoxPackageOutputFolder</RelativeSandBoxPackageOutputFolder>
+    <SandBoxRepoRoot>$(RepoRoot)/$(SandBoxFolderName)/$(LinuxDistrosNeedNativeInstaller)</SandBoxRepoRoot>
+    <NugetCacheFolderName>.nuget</NugetCacheFolderName>
+    <RelativeCompileOutputPath>bin/2</RelativeCompileOutputPath>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <_MirrorOfRepoRootForTools Include="$(RepoRoot)/**/*.*" Exclude="$(RepoRoot)/$(SandBoxFolderName)/**/*.*;$(RepoRoot)/nuget/**/*.*;$(RepoRoot)/bin/**/*.*" />
```

Comment:
Since the sandbox folder is under the bin folder, you can remove `$(RepoRoot)/$(SandBoxFolderName)/**/*.*` from the Exclude.